docs/libGD_logo-1.svg

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why `libGD_logo-1.svg` ?  seems like `logo.svg` would suffice.

Reply:
The very name "logo.svg" says nothing, suggest libGD_logo.svg